Header header
string name
JointDescription[] joints
LinkDescription[] links

# Base position
geometry_msgs/Vector3 base_pos
